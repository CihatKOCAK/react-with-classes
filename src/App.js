import logo from "./logo.svg";
import "./App.css";
import { Component } from "react";
import Counter from "./counter";

class App extends Component {
  render() {
    return (
      <div className="App">
        <a href="https://cihatkocak.netlify.app/">Say Hi to me xd!</a>
        <hr />
        <Counter />
      </div>
    );
  }
}

export default App;
