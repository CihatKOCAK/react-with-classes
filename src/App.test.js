import { render, screen } from "@testing-library/react";
import App from "./App";
import renderer from "react-test-renderer";

describe("snapshot test", () => {
  test("snapshot for App component", () => {
    const renderedComponnt = renderer.create(<App />).toJSON()
    expect(renderedComponnt).toMatchSnapshot();
  })
})
