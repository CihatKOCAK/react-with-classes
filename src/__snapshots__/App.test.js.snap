// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test snapshot for App component 1`] = `
<div
  className="App"
>
  <a
    href="https://cihatkocak.netlify.app/"
  >
    Say Hi to me xd!
  </a>
</div>
`;
